netci.groovy

```diff
@@ -1742,19 +1742,23 @@ combinedScenarios.each { scenario ->
                                         break
                                     }
                                     else {
+                                        // Setup variables to hold emulator folder path and the rootfs mount path
+                                        buildCommands += "armemul_path=/opt/linux-arm-emulator ; armrootfs_mountpath=/opt/linux-arm-emulator-root"
                                         // Unmount previously mounted rootfs and mount the Linux ARM emulator rootfs at /opt/linux-arm-emulator-root/
-                                        buildCommands += "if grep -qs '/opt/linux-arm-emulator-root' /proc/mounts; then sudo umount /opt/linux-arm-emulator-root; fi ; sudo mount /opt/linux-arm-emulator/platform/rootfs-t30.ext4 /opt/linux-arm-emulator-root/"
-                                        // Remove old copy of coreclr and copy the latest version of coreclr
-                                        // This need to be done as it is not possible to clone the repository inside the chroot jail
-                                        buildCommands += "sudo rm -rf /opt/linux-arm-emulator-root/home/coreclr; sudo mkdir /opt/linux-arm-emulator-root/home/coreclr; sudo cp -R ./ /opt/linux-arm-emulator-root/home/coreclr"
-
-                                        // Chroot into the Linux ARM emulator environment and execute the build
-                                        buildCommands += """echo \"Chrooting into Linux ARM emulator environment\"
-                                                            sudo chroot /opt/linux-arm-emulator-root/ /bin/bash -x <<EOF
-                                                            source /dotnet/setenv/setenv_coreclr.sh
-                                                            cd home/coreclr
-                                                            ./build.sh arm clean verbose skipmscorlib
```

Comment:
You are correct, this change breaks the generator. It has been reverted with #5766. @prajwal-aithal please re-issue this PR with fixes.